### ISSUE TITLE
Training stops after first batch of epochs since TensorFlow upgrade

### ISSUE THREAD

**Issue by mzur** [biigle/maia#87]
It seems like the last TensorFlow upgrade broke the training procedure. When started in the default configuration, training just hangs after the first 10 epochs. This is the break where training should continue with the next 10 epochs but a different learning rate.

**Comment by mzur** [comment]
Some people [suggest](https://github.com/keras-team/keras/issues/8595#issuecomment-416111215) to disable multiprocessing with Keras in this case. In a manual test this seems to have worked.
